hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMetrics.java

```diff
@@ -428,10 +430,18 @@ public void incNumListS3Buckets() {
   }
 
   public void incNumListS3BucketsFails() {
-    numBucketOps.incr();
```

Comment:
Unrelated change?

Reply:
Correct. Haven't bothered to file a new jira to do this one-line fix.

`Fails` metric isn't supposed to increase `Ops`.